src/main/webapp/WEB-INF/static/js/trialmanager/germplasm-study-source/germplasmStudySource.js

```diff
@@ -413,6 +413,9 @@
 							resolve: {
 								searchResultDbId: function () {
 									return searchDto.result.searchResultDbId;
+								},
+								searchOrigin: function () {
+									return SEARCH_ORIGIN.MANAGE_STUDY_SOURCE.$name;
```

Comment:
is it possible to resolve the name when creating the constant? (perhaps more safe if we can get a compilation error from thymeleaf when the enum name is changed). Something like:
```javascript
}).constant('SEARCH_ORIGIN', {
  MANAGE_STUDY_SOURCE: /*[[${T(...SearchOrigin).MANAGE_STUDY_SOURCE.name()}]]*/ null
```

Reply:
doesn't matter, renaming the enum should result in a thymeleaf exception anyway